### DISCUSSION TITLE
win版翻页问题

### DISCUSSION THREAD

**Discussion by twtan** [xiaoyaocz/flutter_dmzj#197] [General]
win版用键盘操作翻页每次翻到倒数第3页的时候就会变成直接翻到倒数第1页翻到吐槽页的动画
比如10页的漫画我在第6页想翻第7页，然后就软件就从第6页闪到了第10页然后翻到吐槽页的动画
但是用鼠标拖动又是正常不会出现翻页问题

**Comment by Huang2326** [comment]
附议。最近使用的时候我也发现了这个问题，可能是按键没有除颤，所以就变成了按一下跳很多页面

**Comment by xtc-ketchup** [comment]
用左右按键来翻页的话确实会跳页到最后一页，但是用PageUp和PageDown键翻页不会，用这两个键翻页吧

**Comment by Kumiri** [comment]
未来能更新修复吗？
